package.json

```diff
@@ -10,7 +10,12 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@lukso/lsp-factory.js": "file:../tools-lsp-factory",
```

Comment:
Will be updated as soon as the `lsp-factory` PR was merged, and the package is published.